boards/posix/native_posix/native_posix_defconfig

```diff
@@ -6,3 +6,4 @@ CONFIG_SERIAL=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1000000
 CONFIG_COVERAGE=y
 CONFIG_FAKE_ENTROPY_NATIVE_POSIX=y
+CONFIG_BT_USERCHAN=y
```

Comment:
If we make this part of the default board and require a command-line option this might not work for those not interested in Bluetooth using the native port?

Reply:
I think it should be fine, since the option depends on CONFIG_BT, i.e. the above enabling will never happen if the pre-requisite is not fulfilled. E.g. I just tried building hello_world for native_posix, and not a single CONFIG_BT* option ended up in the final configuration.

Reply:
Also, I'd make the check for the presence of the command line option to happen only when bt_enable() (and consequently drv->open) is called.